src/page-components/unicorns/profile-header/profile-header.module.scss

```
@@ -110,7 +110,8 @@
 	}
 
 	.socialText {
-		margin-left: 16px;
+		margin-left: 0.5em;
```

Let's change this to `rem` - I don't like/trust `em`

```suggestion
		margin-left: 0.5rem;
```